gap/base/recognition.gi

```diff
@@ -212,6 +214,27 @@ InstallGlobalFunction( EmptyRecognitionInfoRecord,
     return ri;
   end );
 
+InstallOtherMethod( RecogNode,
+  "for a group",
+  [ IsGroup ],
+  function(H)
+    return RecogNode(H, false, rec());
+  end );
+
+InstallOtherMethod( RecogNode,
+  "for a group and a boolean",
+  [ IsGroup, IsBool ],
+  function(H, projective)
+    return RecogNode(H, projective, rec());
+  end );
+
+InstallOtherMethod( RecogNode,
+  "for backwards compatibility",
```

Comment:
What are you trying to be backwards compatible with, though? is there still code in the repo that calls it like that? If so, can't we just update that code?

Other than that: any external code won't be calling `RecogNode`, as it didn't exist before, right? It'll call `EmptyRecognitionInfoRecord`, so that code needs to be adjusted anyway. Or perhaps we can retain `EmptyRecognitionInfoRecord` in a new file `gap/obsolete.gi`; but then that backwards compatible function can take of reordering the arguments.

Perhaps I am missing something, though?

Reply:
> Perhaps I am missing something, though?

Nope, that makes perfect sense, thanks! :+1:

Reply:
I'm going with this:
> Or perhaps we can retain EmptyRecognitionInfoRecord in a new file gap/obsolete.gi; but then that backwards compatible function can take of reordering the arguments.